cockroachcloud/frequently-asked-questions.md

```diff
@@ -84,22 +89,19 @@ See the [Security Overview page](../{{site.current_cloud_version}}/security-refe
 
 ### Is encryption-at-rest enabled on {{ site.data.products.dedicated }}?
 
-All data in {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} is encrypted-at-rest by your chosen infrastructre-as-a-service provider, Google Cloud Platform (GCP) or Amazon Web Services (AWS), at the infrastructure level.
+All data in {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} is encrypted-at-rest by your chosen infrastructre-as-a-service provider, Google Cloud Platform (GCP), Amazon Web Services (AWS), or Microsoft Azure (limited access), at the infrastructure level.
 
 {{site.data.alerts.callout_info}}
-{{ site.data.products.serverless }} and {{ site.data.products.dedicated }} users delegate responsibility for encryption-at-rest to the cloud provider. Hence, CockroachDB's proprietary storage-layer encryption-at-rest functionality is currently only available with an Enterprise license and is not currently available to users of {{ site.data.products.serverless }} or {{ site.data.products.dedicated }}.
+{{ site.data.products.serverless }} and {{ site.data.products.dedicated }} users delegate responsibility for encryption-at-rest to the cloud provider. CockroachDB's proprietary storage-layer encryption-at-rest functionality is currently only available with an Enterprise license and is not currently available to users of {{ site.data.products.serverless }} or {{ site.data.products.dedicated }}.
```

Comment:
This needs change. "CockroachDB's proprietary storage-layer encryption-at-rest functionality" is basically CMEK which is available for dedicated on AWS/GCP.